Study name shown with /studies is wrong

forus opened the pull request:
Also rename name field to id one in Study

Commit:
Study name shown with /studies is wrong

Also rename name field to id one in Study

Commit:
Study name shown with /studies is wrong

Also rename name field to id one in Study

cataphract commented on grails-app/services/org.transmartproject/rest/StudyLoadingService.groovy:
```diff
@@ -129,11 +129,11 @@ class StudyLoadingService {
              * for consistency?)
              */
-            studyName = term.name
+            studyId = term.name
```
This looks wrong. You're using the term name as id, but the whole point of your changes were to decouple the two things. You'll have to call `term.study.id`.

forus replied:
Yes, good catch. Thanks!

Commit:
Merge branch 'issues/study-name-is-wrong' of github.com:thehyve/transmart-rest-api into issues/study-name-is-wrong

Commit:
Modify API documentation

- studyName -> studyId
- rename file to README.md so it'll be displayed in first page of github

Commit:
Get id from study insted of term name
